src/toolkit/matl_buy_policy.cc

```diff
@@ -158,17 +157,10 @@ std::set<RequestPortfolio<Material>::Ptr> MatlBuyPolicy::GetMatlRequests() {
       Material::Ptr m = Material::CreateUntracked(req_amt, d.comp);
       grps[i].push_back(port->AddRequest(m, this, commod, d.pref, excl));
     }
-
-    // if there's more than one commodity, then make them mutual
-    if (grps.size() > 1) {
-      std::map<int, std::vector<Request<Material>*> >::iterator grpit;
-      for (grpit = grps.begin(); grpit != grps.end(); ++grpit) {
-        port->AddMutualReqs(grpit->second);
-      }
-    }
+    port->AddMutualReqs(grps[i]);
```

Comment:
The previous behavior seems to have caused this. Not sure why it was implemented in the first place. I imagine there is a good reason but I can't think of one at the moment.